Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -6,34 +6,39 @@ public enum ActorStates
 {
 	Grounded = 0,
 	Jumping,
+	Falling,
 	Rolling,
 	Climbing
 }
 
 public class ActorPhysics : ActorComponent
 {
+	private new Transform transform;
+	private new Rigidbody rigidbody;
+
 	// States
+	#region States
 	public delegate void ActorStateMethod();
 
+	[ReadOnlyAttribute, SerializeField]
```

Comment:
Yeah, that should actually be just `ReadOnly`, I'm surprised having `ReadOnlyAttribute` didn't cause a build error.